### DISCUSSION TITLE
Docs not up-to-date?

### DISCUSSION THREAD

**Discussion by petzi53** [Zettlr/Zettlr#3605] [Q&A] [answered]
I am using Zettlr version 2.3.0 for the Intel Mac. I noticed that the documentation is different in German and English. Take for instance the [tag manager](https://docs.zettlr.com/en/reference/settings/#tag-manager) resp. [schlagworte verwalten](https://docs.zettlr.com/de/reference/settings/#schlagworte-verwalten). The English version does not conform to my app version. So I assumed that the German version is the recent and correct one. 

But this is incorrect as I also found differences in German explanations and screenshots with my installation. For instance, the [Video Introduction 001](https://www.youtube.com/watch?v=dQ_tpVtLX8k&list=PL2ydvDwV-1u5ncnkxQWZhbHQdwpY2fx5f&index=1) talks about RMarkdown files (.Rmd), and the German documentation shows a screenshot of the advanced preferences where I can enable RMarkdown support. But my window for preferences is structured differently, and I could not find a switch for turning on RMarkdown support.

**Comment by nathanlesage** [comment, marked as answer]
First, apologies that yes, all documentation is a tad outdated at the moment.

The main problem is that I didn't have time to update the documentation in recent time, and community engagement has been lacking a bit.

In general, the English documentation is the one I update when I get to it, so this should serve as a reference documentation. Most translations are the work of community members.

**Reply by nathanlesage** [reply]
With regard to the specific issue of Rmd-files: These are now supported by default, so the switch is not necessary anymore (and I think it was never really necessary!)

**Reply by petzi53** [reply]
Thanking you for answering. 

- **Concerning Rmd-files**: I have to check it out (again). I believe I tried it already but experienced some issues. Can't remember what the problem was. I will come back to you after I have rechecked it.
- **Concerning documentation**: Frankly, I do not know what the advantages of Zettlr are in contrast to other markdown editors. Maybe you could point them out in the planned revision of the documentation?
